django/contrib/contenttypes/fields.py

```diff
@@ -253,6 +253,18 @@ def __set__(self, instance, value):
         setattr(instance, self.cache_attr, value)
 
 
+class GenericRel(ForeignObjectRel):
+    def __init__(self, field, to, related_name=None, related_query_name=None, limit_choices_to=None):
+
```

Comment:
I'd personally omit the newlines you've added before super (don't recall this is a pattern we've used elsewhere)